services/state/roster/roster.proto

```diff
@@ -88,10 +105,6 @@ message RosterEntry {
      * then endpoints in this list MAY supply either IP address or FQDN, but
      * SHALL NOT supply both values for the same endpoint.<br/>
      * This list SHALL NOT be empty.<br/>
-     * This list SHALL NOT contain more than `10` entries.<br/>
-     * The first two entries in this list SHALL be the endpoints published to
-     * all consensus nodes.<br/>
-     * All other entries SHALL be reserved for future use.
```

Comment:
These lines are a requirement in the Node definition.  Are we expecting something different now?

Reply:
>The first two entries in this list SHALL be the endpoints published to all consensus nodes
Richard pushed back on the correctness of this on the proposal PR so i removed it.
And as for the size, it's something that's defined in the code and I believe should be validated right where it's defined. The prototype definition itself has no facility to enforce it.

Reply:
Three points:

The specification should still be expressing such limitations; else how does a consumer of block stream (this is a public API via block stream) know what those limitations are?

Out protobuf definitions _may_ be extended with proto-validate or similar support when PBJ is able to handle that; these specifications are _hoped_ to support doing so with minimal need to re-read the full codebase.

If the requirement is _actually_ incorrect, then that should be discussed with services as well, and the change (including to specification in the node create and update transactions) added to any HIP related to adopting the Roster.

The primary purpose of specification is to enable consumers of state information to correctly consume the data, and also support any sufficiently motivated entity to create an independent, and correct, consensus node implementation.

Any code enforced limitations that are relevant _should_ be included.

Reply:
1. The specification can refer to configuration, but the critical element is that it is currently specified in the node create and node update transactions, so it's a HIP to change it there.  If the limitation is variable (I don't actually know the source of the limit), then we probably need a cross-team conversation to clarify that.
1. The line can be removed, if it's not accurate. I believe the original line came from a platform "this is what we currently do" requirement.  If that's changed, then, again, we probably need to make sure services is aware and that all of the associated documentation (including code) is updated.

Reply:
noted, I'll check with services.